playbooks/kubevirt-web-ui/README.md

```diff
@@ -39,9 +39,9 @@ Please check `playbooks/kubevirt-web-ui/inventory_example.ini` for an example.
 
 ### Invocation examples
 ```
-ansible-playbook -i your_inventory_file.ini playbooks/kubevirt-web-ui/config.yml -e "apb_action=provision platform=openshift registry_url=quay.io registry_namespace=kubevirt docker_tag=1.3" # to mimic kubevirt-apb flow
+ansible-playbook -i your_inventory_file.ini playbooks/kubevirt-web-ui/config.yml -e "apb_action=provision platform=openshift registry_url=quay.io registry_namespace=kubevirt docker_tag=1.4" # to mimic kubevirt-apb flow
```

Comment:
Can someone plz ACK this explicitly, just to confirm and we be double sure.

Reply:
@SchSeba be that someone, please!

Reply:
Please use tag `v1.4.0` as requested by @rmohr  to allow z-stream updates.

Reply:
I felt something was not right, hence asked for an explicit confirmation, thanks.